NEWS

```diff
@@ -11,6 +11,171 @@
 ### Features:
 ### Bugfixes:
 
+## 1.13.0 (May 19, 2022)
+#### Features
+##### Core
+* Added new objects to VFS: local and remote address of enpoint, statistics of ucp_ep_create success/failure, failed/destroyed endpoints
+* Added support of UCX static libraries
+* Added profiling for rkey management routines
+* Enabled by default PCIe relaxed order for AMD CPUs
+#### UCP
+* Added API to pass pre-registered memory handle to UCP operations
+* Added implementation of AM rendezvous protocol
+* Added 2-stage pipeline rendezvous protocol for GPU
+* Added support of fragment mem_type for v1 pipeline proto, disabled by default
+* Added active message support for proto v2
+* Added UCP memory registration cache
+* Improved adaptive progress - deactivate iface when discard/cleanup p2p lanes
+* Added support of user memh in proto_v1
+* Added support selecting local address when creating a client endpoint
+* Added option to limit GPUDirectRDMA size in rendezvous protocol, UCX_RNDV_MEMTYPE_DIRECT_SIZE
+* Deprecated UCX_SOCKADDR_AUX_TLS configuration parameter
+#### UCT
+* Introduced API uct_md_mkey_pack_v2
+* Introduced interfaces features API
```

Comment:
I think yes, @gleon99 could you pls comment what did you mean in ed22d5cd1e1f30ef939578eee163d4f64cc93586?

Reply:
This API allows to initialize only specific UCT iface features, rather than supporting everything - according to the UCP context config features requested by the user.

Currently it is only used to avoid initializing `PUT` on `DC` when it is not needed, to workaround some FW issue. In the future we can add more useful scenarios there.

Reply:
@gleon99 could you pls suggest better wording for NEWS?

Reply:
How about:
`* Introduced UCT iface features API` ?

Reply:
done